src/solvers/exponentiate.jl

```diff
@@ -0,0 +1,48 @@
+function exponentiate_updater(
+  init;
+  psi_ref!,
+  PH_ref!,
+  outputlevel,
+  which_sweep,
+  region_updates,
+  which_region_update,
+  region_kwargs,
+  updater_kwargs,
```

Comment:
Is the plan to combine these into `updater_kwargs`?

Reply:
`updater_kwargs` are `updater` specific kwargs, while `region_args` are among the things that we expose to all updaters. in principle, we can nest the region_args into updater_kwargs in the call to `region_update` but I am not sure if that's actually preferable.

Reply:
Hmm ok, I just have found it hard to keep track of the logic of why certain keyword arguments are bundled in certain ways, how they will be used, etc.

For example, from the perspective of this function, the only argument I can see that is being used here from `region_kwargs` is `time_step`, which I don't think is really any different conceptually from the arguments being passed in `updater_kwargs` (it's just another thing being used by the solver/updater). So it makes sense to me to just bundle those together in one flat `NamedTuple` called `updater_kwargs`.

Reply:
As discussed, these will eventually be bundled together in an upcoming PR.